src/nupic/engine/Network.hpp

```diff
@@ -250,7 +286,7 @@ class Link;
    *
    * @returns A Collection of Link objects in the network
    */
-  Collection<std::shared_ptr<Link>> getLinks();
+  std::vector<std::shared_ptr<Link>> getLinks() const;
```

Comment:
Can th4e Collection class be removed?

Reply:
> Can the Collection class be removed?

Would love to....but see the previous API breaking discussion.